VersionControlUI/styles/item/HighlightOverlay/delete

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE EnvisionVisualizationStyle>
+<style prototypes="item/DeclarativeItemBase/default">
```

Comment:
`prototypes` is a list of one or more file names that contain properties. The search for a file is done first relative to the current file, and if that fails, then starting from the `styles` directory.

Here you can use just the name of another file in this directory. When I say this directory, I mean the HighlightOverlay directory as it will exist once you run `make install`. So also files that will be coppied here from other plug-ins can be used (in particular `default` from VisualizationBase)